### PR TITLE
Unexclude TestHandshake

### PR DESCRIPTION
**Pull request by babsingh**
TestHandshake has been fixed by

- https://github.com/eclipse-openj9/openj9/pull/19167
- https://github.com/eclipse-openj9/openj9/pull/19412

Related: https://github.com/eclipse-openj9/openj9/issues/13211

### CODE REVIEW

**Comment by babsingh** [comment]
fyi @pshipton

File: openjdk/excludes/ProblemList_openjdk16-openj9.txt [hunk 1]
```diff
@@ -391,7 +391,6 @@ vm/verifier/VerifyProtectedConstructor.java	https://github.com/eclipse-openj9/op
 ############################################################################
 
 # jdk_foreign
-java/foreign/TestHandshake.java https://github.com/eclipse-openj9/openj9/issues/13211 generic-all
```
Comment (pshipton) [inline openjdk/excludes/ProblemList_openjdk16-openj9.txt:394]: I don't think we should be modifying versions which are out of support and not getting any updates. 
i.e. 16, 18, 19, 20

Comment (babsingh) [reply openjdk/excludes/ProblemList_openjdk16-openj9.txt]: Updated.

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**